recipe/meta.yaml

```
@@ -28,12 +29,24 @@ requirements:
     - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy # bounds through run_exports
```
This yields a higher range than pin_compatible.